README.md

```diff
@@ -32,6 +32,10 @@ Clone the repository
 
   - Production: `$ fvm flutter run --flavor production`
 
+- Generate assets folder
+  
+  - `$ flutter packages pub run build_runner build`
```

Comment:
Please add `fvm` as a prefix to match with the above commands? 😄 
```suggestion
  - `$ fvm flutter packages pub run build_runner build`
```

Reply:
even there is a command to execute the assets builder, should we commit those generated assets files into git? 😂 I think I agree to commit them now @sleepylee @manh-t 🤔 

@doannimble if we don't commit them, we should put `gen` dir into gitignore 🤓

Reply:
@luongvo yeah, I agree to commit them into git 🛴

Reply:
@luongvo @manh-t I don't think we need to commit `gen` folder into the repository. It is unnecessary and making potential conflict on development.
P/S : I would like to add it on `.gitigonre`

Reply:
@doannimble IMO, the `gen` assets are rarely changed unless we are adding 2 files with the same name. The potential conflict could be from `generated json, freezed files` I have the same concern. But when I'm resolving the conflict, I always ignore them and rebuild them all. So it's not a big deal to me.
How do you think? @luongvo @sleepylee @markgravity I can make an RFC for this.

Reply:
Except we have another way to generate them automatically when launching the app after switching between branches, I think we should commit them 🤓

Reply:
@manh-t Let say, now we have one image name `number1` on master. On my branch, I add an image name `number2`. On @luongvo 's branch, he adds the `number3`. My branch is merged first. When @luongvo `pull` for update, he will be got conflict in `assets.gen.dart`.
Spend more time to resolve conflict (event just click to ignore) is not good. Ignore this, we will be more freely. Is it not better? I'm willing to listen from the others too.

Reply:
@sleepylee our discussion here 💁‍♂️

Reply:
with `$ fvm flutter packages pub run build_runner build --delete-conflicting-outputs` you will never encounter issue with build conflict 💁‍♂️  maybe just a small git conflict but oftentimes, they can be resolved by adding both.

I'm leaning more toward the speed for the CI to reduce the gen time + the convenience of a new developer can pull and run the project without executing many commands at the beginning.
Flutters ecosystem has many static code generators afaik, so not only gen, later we will have to exclude `.g.dart` file as well? 

Another point to consider: it is not pointed directly from Flutter docs that we have to ignore it https://dart.dev/guides/libraries/private-files.

Lastly, if you do a quick search, this topic is very debatable everywhere, in every teams... so let's do it in democracy way: let's vote. 🙋

Reply:
I agree to commit them 👉  1️⃣

Reply:
https://stackoverflow.com/questions/53513860/flutter-which-folder-not-to-commit-to-svn 

another common pattern for files/dirs that should be ignored in my experience is that they are **generated outside of the main codebase**, in our case, it is generated in `lib/` which is like our `src/` in an Android project; not in `builds/generated/`...so these are quite different. 1 is boilerplate code, the other generates the actual enough code to grant us access to those assets path.

Reply:
I agree to commit them. Btw, let me create the vote for us 🙋‍♂️

Reply:
@sleepylee Okay, so I agree that we should keep all files of our main codebase. Just ignored the generated outside.
So the final decision is commit them 👌

Reply:
oh, so we don't need an RFC anymore if all members agree with this. 🤔 
@markgravity we need your input as well 😄

Reply:
🙌  don't hesitate if you guys found anything different in the future, these things can be easily changed if we have to 🙆‍♂️  let's get our first step done 🏁 🚴‍♂️  🚴 🚴‍♂️  🚴 🚴‍♂️  🚴